core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1680,6 +1680,13 @@ package object config {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("32k")
 
+  private[spark] val IO_COMPRESSION_ZSTD_ENABLE_BUFFERPOOL =
+    ConfigBuilder("spark.io.compression.zstd.enableBufferPool")
```

Comment:
maybe `spark.io.compression.zstd.bufferPool.enabled` :-)?

Reply:
Thank you for review. I cannot make `bufferPool` as a new namespace because we have only one configuration here `enableBufferPool`.

Reply:
Looks like postfix `.enabled` is more common in this file though.

```
    ConfigBuilder("spark.driver.log.persistToDfs.enabled")
  private[spark] val EVENT_LOG_ENABLED = ConfigBuilder("spark.eventLog.enabled")
    ConfigBuilder("spark.eventLog.logBlockUpdates.enabled")
    ConfigBuilder("spark.eventLog.erasureCoding.enabled")
    ConfigBuilder("spark.eventLog.longForm.enabled")
    ConfigBuilder("spark.eventLog.rolling.enabled")
    ConfigBuilder("spark.executor.processTreeMetrics.enabled")
  private[spark] val MEMORY_OFFHEAP_ENABLED = ConfigBuilder("spark.memory.offHeap.enabled")
      "If off-heap memory use is enabled, then spark.memory.offHeap.size must be positive.")
      "accordingly. This must be set to a positive value when spark.memory.offHeap.enabled=true.")
    ConfigBuilder("spark.storage.decommission.enabled")
    ConfigBuilder("spark.storage.decommission.shuffleBlocks.enabled")
    ConfigBuilder("spark.storage.decommission.rddBlocks.enabled")
    ConfigBuilder("spark.dynamicAllocation.enabled")
    ConfigBuilder("spark.dynamicAllocation.shuffleTracking.enabled")
    ConfigBuilder("spark.shuffle.service.enabled")
    ConfigBuilder("spark.shuffle.service.db.enabled")
    ConfigBuilder("spark.task.reaper.enabled")
    ConfigBuilder("spark.excludeOnFailure.enabled")
      .withAlternative("spark.blacklist.enabled")
    ConfigBuilder("spark.excludeOnFailure.application.fetchFailure.enabled")
      .withAlternative("spark.blacklist.application.fetchFailure.enabled")
    ConfigBuilder("spark.metrics.executorMetricsSource.enabled")
    ConfigBuilder("spark.metrics.staticSources.enabled")
  private[spark] val IO_ENCRYPTION_ENABLED = ConfigBuilder("spark.io.encryption.enabled")
    ConfigBuilder("spark.authenticate.enableSaslEncryption")
    ConfigBuilder("spark.shuffle.reduceLocality.enabled")
    ConfigBuilder("spark.unsafe.sorter.spill.read.ahead.enabled")
    ConfigBuilder("spark.executor.logs.rolling.enableCompression")
  private[spark] val MASTER_REST_SERVER_ENABLED = ConfigBuilder("spark.master.rest.enabled")
    ConfigBuilder("spark.decommission.enabled")
    ConfigBuilder("spark.shuffle.push.enabled")
```

I think we have used `.enabled` postfix  to indicate a boolean configuration, and setting it to `true` is enabling it.

Reply:
.. but probably it's not a big deal since there's already inconsistency there ..

Reply:
I don't have a strong preference, but our policy below says naming `featureName.enabled` is a best practice... as @HyukjinKwon suggested above. So, I tend to follow it in most cases;

https://github.com/apache/spark/blob/89bf2afb3337a44f34009a36cae16dd0ff86b353/core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala#L33-L40

Reply:
So, `.enabled` can make a namespace? I'm fine to make it as `.enabled`. I'll update the PR soon.

Reply:
Thanks @dongjoon-hyun.